pages/common/pdftocairo.md

```diff
@@ -0,0 +1,28 @@
+# pdftocairo
+
+> Converts PDF files to PNG/JPEG/TIFF/PDF/PS/EPS/SVG using cairo.
+> More information: <https://poppler.freedesktop.org>.
+
+- Convert a PDF file to JPEG:
+
+`pdftocairo {{path/to/file}}.pdf -jpeg`
+
+- Convert to PDF expanding the output to fill the paper:
+
+`pdftocairo {{path/to/file}}.pdf {{output.pdf}} -pdf -expand`
+
+- Convert to SVG specifying the first/last page to convert:
+
+`pdftocairo {{path/to/file}}.pdf {{output.svg}} -svg -f {{number}} -l {{number}}`
```

Comment:
The two `number`s should have different names, e.g. `first_page` and `last_page`.